.github/workflows/ci.yml

```diff
@@ -38,6 +41,36 @@ jobs:
         with:
           name: logs-${{ matrix.machine }}
           path: build_data/
+      - run: brew install coreutils
+        if: ${{ runner.os == 'macOS' }}
+      - name: Create test directory
+        run: sudo mkdir -p ${{ matrix.log-dir }} && sudo chmod a+rwx ${{ matrix.log-dir }}  # workaround before next release
```

Comment:
The release with fix for creating whole log structure on linux/macosx is already out https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/CHANGELOG.md#040-beta1

Is it really workaround or we have some bug in product?

Reply:
Well, it doesn't work without this step...
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1703/commits/da5db9c74f806e1bb15042720cc64a397a43e765

Reply:
Why? Lack of privileges or bug in product?

Reply:
When I was testing that locally in linux container it worked perfectly fine without manually creating the directory and adding permissions, so I would lean towards lack of privilege.